kogito-kafka-quickstart-springboot/README.md

```diff
@@ -71,8 +129,21 @@ To make use of this application it is as simple as putting a message on `travell
 
 this will then trigger the successful processing of the traveller and put another message on `processedtravellers` topic.
 
+* To examine ProcessedTravellers topic and verify it was auto sent
```

Comment:
I would put this part before the `Send message`, so we can then send 2 different messages

Reply:
Fixed @radtriste